ibm/data_source_ibm_pi_placement_group.go

```diff
@@ -0,0 +1,79 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package ibm
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/helpers"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func dataSourceIBMPIPlacementGroup() *schema.Resource {
+
+	return &schema.Resource{
+		ReadContext: dataSourceIBMPIPlacementGroupRead,
+		Schema: map[string]*schema.Schema{
+			helpers.PIPlacementGroupID: {
```

Comment:
Isn't `PIPlacementGroupName` as required and `PIPlacementGroupID` as computed?

Reply:
I made this suggested change.